Formula/redis@6.2.rb

```diff
@@ -0,0 +1,45 @@
+class RedisAT62 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "https://redis.io/"
+  url "https://download.redis.io/releases/redis-6.2.7.tar.gz"
+  sha256 "b7a79cc3b46d3c6eb52fa37dde34a4a60824079ebdfb3abfbbfa035947c55319"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url "https://download.redis.io/releases/"
+    regex(/href=.*?redis[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "openssl@1.1"
```

Comment:
Will it work with `openssl@3`?

Reply:
Yes, this builds and tests successfully with `openssl@3`.

As [redis.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/redis.rb) uses `openssl@1.1`, what would be the preferred approach be? Should I update this formula to the newer version and then raise a separate pull request to update the other formula so that there is not a mismatch?

Reply:
Yeah, sounds good

Reply:
Thanks. https://github.com/Homebrew/homebrew-core/pull/102348 has been created to update `redis.rb`.